tests/vexflow_test_helpers.ts

```diff
@@ -90,6 +90,11 @@ if (!global.$) {
 
 export type TestFunction = (options: TestOptions, contextBuilder: ContextBuilder) => void;
 
+export type RunOptions = {
+  jobs: number;
+  job: number;
```

Comment:
This is a bit confusing, maybe `numJobs` and `jobIndex`? (sorry, I know it's all over the code, but it makes it hard to tell what it's for.)

Reply:
- numJobs is length of VexFlowTests.tests.
  - to map tests to available cpus split tests into jobs.
    - Since flow.html contains multiple canvases and svg, and since the number of tests is likely to increase in the future, we will divide it into 10 parts for fear of exhausting resources. : https://github.com/h-sug1no/vexflow/blob/0a8eec238857cf52d82cc711a9fb61aa230cf756/tools/generate_images.js#L120
- VexFlowTests.run() interprets jobs and job arguments as follows
  - https://github.com/h-sug1no/vexflow/blob/0a8eec238857cf52d82cc711a9fb61aa230cf756/tests/vexflow_test_helpers.ts#L160
    - This is the implementation because we believe that tests should not depend on the order of execution.

I hope that explains it...